proto/regen/ecocredit/v1alpha1/tx.proto

```diff
@@ -274,4 +280,41 @@ message MsgUpdateClassMetadata {
 }
 
 // MsgUpdateClassMetadataResponse is the MsgUpdateClassMetadata response type.
-message MsgUpdateClassMetadataResponse {}
\ No newline at end of file
+message MsgUpdateClassMetadataResponse {}
+
+message MsgSell {
+  string owner = 1;
+  repeated Order orders = 2;
+
+  message Order {
+    string batch_denom = 1;
+    string quantity = 2;
+    string ask_denom = 3;
+    string ask_price = 4;
+    bool dont_auto_retire = 5;
```

Comment:
I imagine that long-term we would want auto retire disabled by default. If that's the case, should this bool be flipped? e.g. set this field to `bool auto_retire_enabled = 5;`?

Reply:
Hmmm... my thinking is we want it on by default - thus the awkward "dont_auto_retire"

Reply:
Can we use `no_` prefix instead of `dont`?

Reply:
don't have a strong preference. `disable_auto_retire` would be most clear i think

Reply:
I prefer `disable_auto_retire` too